lib/py/reddit_edgecontext/__init__.py

```diff
@@ -436,9 +436,9 @@ def name(self) -> str:
             service.
 
         """
-        subject = self.authentication_token.subject
-        if subject is None or subject == "":
```

Comment:
from my understanding, the bug is that we could just chop the first 8 characters from the t2 id to return here, is that correct?

Reply:
this is for service auth, so we should be returning a name only if it's prefixed by ('service/')